clang/include/clang/AST/Decl.h

```diff
@@ -2348,6 +2348,10 @@ class FunctionDecl : public DeclaratorDecl,
   /// entry point into an executable program.
   bool isMain() const;
 
+  /// CHEERP: Determines whether this function is "webMain", which is the
+  /// entry point into an executable program.
+  bool isWebMain() const;
```

Comment:
It seems simpler and more maintainable to hardcode the mangled webmain symbol in the start function.